sonata.core

```diff
@@ -30,6 +31,7 @@ filesets:
       - lowrisc:dv_dpi_c:usbdpi:0.1
       - lowrisc:dv_dpi_sv:usbdpi:0.1
     files:
+      - rtl/system/rs485_ctrl.sv: { file_type: systemVerilogSource }
```

Comment:
Why do you need to add it here again when it's already in files_sonata?

Reply:
`files_verilator` is used for the verilator build and `files_sonata` is only used for the FPGA build.

`rs485_ctrl` is a little awkward as really it'd sit in `sonata_system` but because of the way we've got the pinmux setup (and it needs to sit on top of certain pins from pinmux) it's neater to integrate into `top_sonata` and so it's separately included in top_verilator`